src/static/css/cellml.css

```diff
@@ -115,6 +115,14 @@ div[class^="issue"], div[class*=" issue"] > ol {
     font-family: "Lato","proxima-nova","Helvetica Neue",Arial,sans-serif;
 }
 
+.heading1 {
+    color: #444444;
+    font-family: "Roboto Slab","ff-tisa-web-pro","Georgia",Arial,sans-serif;
+    font-size:200%;
```

Comment:
space after colon

Reply:
Actually, through the whole file ... the only reason I notice is because I fixed them all today in another PR :) !